LICENSE

```diff
@@ -1,6 +1,6 @@
 BSD 3-Clause License
 
-Copyright (c) 2018, The Scikit-HEP Project
+Copyright (c) 2018-2019, The particle developers.
```

Comment:
I think The SciKit-HEP developers would be fine. (And Particle should be capitalized if used)

Reply:
That's "a bit" too much to give everyone contributing to any Scikit-HEP org package the copyright hold, I find. I will capitalise "particle", sure.

Reply:
Following the comments from Hans for boost-histogram, would you prefer to copyright to us two only, just as there? That's OK for such a well-defined package/project done 100% by us anyway.